src/ObjLoading/Game/T6/BSP/BSPUtil.cpp

```diff
@@ -0,0 +1,173 @@
+#pragma once
+
+#include "BSPUtil.h"
+
+#include <cmath>
+#include <format>
+
+namespace BSP
+{
+    std::string BSPUtil::getFileNameForBSPAsset(std::string& assetName)
+    {
+        return std::format("BSP/{}", assetName);
+    }
+
+    vec3_t BSPUtil::convertToBO2Coords(vec3_t& coordinate)
+    {
+        vec3_t result;
+        result.x = coordinate.x;
+        result.y = -coordinate.z;
+        result.z = coordinate.y;
+        return result;
+    }
+
+    vec3_t BSPUtil::convertFromBO2Coords(vec3_t& coordinate)
+    {
+        vec3_t result;
+        result.x = coordinate.x;
+        result.y = coordinate.z;
+        result.z = -coordinate.y;
+        return result;
+    }
+
+    void BSPUtil::updateAABB(vec3_t& newAABBMins, vec3_t& newAABBMaxs, vec3_t& AABBMins, vec3_t& AABBMaxs)
+    {
+        if (AABBMins.x > newAABBMins.x)
+            AABBMins.x = newAABBMins.x;
+
+        if (newAABBMaxs.x > AABBMaxs.x)
+            AABBMaxs.x = newAABBMaxs.x;
+
+        if (AABBMins.y > newAABBMins.y)
+            AABBMins.y = newAABBMins.y;
+
+        if (newAABBMaxs.y > AABBMaxs.y)
+            AABBMaxs.y = newAABBMaxs.y;
+
+        if (AABBMins.z > newAABBMins.z)
+            AABBMins.z = newAABBMins.z;
+
+        if (newAABBMaxs.z > AABBMaxs.z)
+            AABBMaxs.z = newAABBMaxs.z;
```

Comment:
These can be changed to
```cpp
        AABBMins.x = std::min(AABBMins.x, newAABBMins.x);
        AABBMaxs.x = std::max(AABBMaxs.x, newAABBMaxs.x);

        AABBMins.y = std::min(AABBMins.y, newAABBMins.y);
        AABBMaxs.y = std::max(AABBMaxs.y, newAABBMaxs.y);

        AABBMins.z = std::min(AABBMins.z, newAABBMins.z);
        AABBMaxs.z = std::max(AABBMaxs.z, newAABBMaxs.z);
```

which is a bit more readable imo 😅 

Adjusted in a8456f88a49fae340446833234f33df1c2274e5b